src/main/java/io/jenkins/plugins/jfrog/configuration/JFrogPlatformBuilder.java

```diff
@@ -30,6 +30,8 @@
  * @author gail
  */
 public class JFrogPlatformBuilder extends GlobalConfiguration {
+    private static final String UNSAFE_HTTP_ERROR = "HTTP connections to the JFrog platform services are not allowed. " +
```

Comment:
HTTP connections to the JFrog platform services are not allowed.
-->
HTTP (non HTTPS) connections to the JFrog platform services are not allowed by default.